src/instructlab/cli/model/evaluate.py

```diff
@@ -235,9 +250,34 @@ def evaluate(
 ) -> None:
     """Evaluates a trained model"""
     try:
-        judge_model, output_dir = set_benchmark_specific_vars(
-            ctx, benchmark, judge_model, output_dir
-        )
+        if base_model_id:
+            # select the base model configuration and the system prompt to use for the evaluation
+            base_model_cfg = resolve_model_id(base_model_id, ctx.obj.config.models)
+            if not base_model_cfg:
+                raise click.exceptions.BadParameter(
+                    f"Base model with ID '{base_model_id}' not found in the configuration."
+                )
+            base_model = base_model_cfg.path
+            system_prompt = base_model_cfg.system_prompt
+            logger.debug(
+                "detected `--base-model-id`, selecting custom model from config list."
+            )
+
+        if judge_model_id:
+            judge_model_cfg = resolve_model_id(judge_model_id, ctx.obj.config.models)
+            if not judge_model_cfg:
+                raise click.exceptions.BadParameter(
+                    f"Base model with ID '{judge_model_id}' not found in the configuration."
+                )
+
+            judge_model = judge_model_cfg.path
+            logger.debug(
+                "detected `--judge-model-id`, selecting custom model from config list."
+            )
+        else:
+            judge_model, output_dir = set_benchmark_specific_vars(
```

Comment:
@jaideepr97 Should these be separate cases? 

It seems like `output_dir` is being set in the fallback case from `set_benchmark_specific_vars`, but when the IDs are passed it seems like we bypass the benchmark-specific settings.